addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -202,5 +206,4 @@
             <field name="res_model">project.project</field>
             <field name="view_mode">qweb</field>
         </record>
-
```

Comment:
Not sure you have to remove this line as it is not always the case in odoo and there is absolutely no guidelines on that. So if a line is not part of a modified block don't change things.